build.release.yaml

```diff
@@ -0,0 +1,19 @@
+# See https://github.com/dart-lang/build/tree/master/build_web_compilers#configuration
+# Matches previous configuration in pubspec.yaml - transformers - $dart2js
+targets:
+  $default:
+    builders:
+      build_web_compilers|entrypoint:
+        # These are globs for the entrypoints you want to compile.
+        generate_for:
+        - web/**.dart
+        options:
+          compiler: dart2js
+          dart2js_args:
+            # Align with the default configuration of dart2js in `pub build`
```

Comment:
does this align with `pub build` defaults? I wouldn't think some of these would be the default...

Reply:
Ooops... Yeah. Now they don't. Will fix...